### PR TITLE
MINOR: standardize rebalance related logging for easy discovery & debugging

### PR DESCRIPTION
**Pull request by ableegoldman**
Some minor logging adjustments to standardize the grammar of rebalance related messages and make it easy to query the logs for quick debugging results

### CODE REVIEW

**Comment by ableegoldman** [comment]
@guozhangwang

**Comment by ableegoldman** [comment]
Unrelated `kafka.api.ConsumerBounceTest failed`

File: clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java [hunk 1]
```diff
@@ -598,11 +598,12 @@ public void handle(JoinGroupResponse joinResponse, RequestFuture<ByteBuffer> fut
                     }
                 }
             } else if (error == Errors.COORDINATOR_LOAD_IN_PROGRESS) {
-                log.debug("Attempt to join group rejected since coordinator {} is loading the group.", coordinator());
+                log.debug("JoinGroup failed: Coordinator {} is loading the group.", coordinator());
                 // backoff and retry
                 future.raise(error);
             } else if (error == Errors.UNKNOWN_MEMBER_ID) {
-                log.debug("Attempt to join group failed due to unknown member id with {}.", sentGeneration);
+                log.info("JoinGroup failed: {} Need to re-join the group. Sent generation was {}",
```
Comment (guozhangwang) [inline clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java:605]: Why promote it from debug to info while leaving others debug?

File: clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java [hunk 2]
```diff
@@ -769,24 +771,27 @@ public void handle(SyncGroupResponse syncResponse,
...
+                    log.info("SyncGroup failed: {} Need to re-join the group. Sent generation was {}",
+                            error.message(), sentGeneration);
                     if (generationUnchanged())
                         resetGenerationOnResponseError(ApiKeys.SYNC_GROUP, error);
 
                     future.raise(error);
                 } else if (error == Errors.COORDINATOR_NOT_AVAILABLE
                         || error == Errors.NOT_COORDINATOR) {
-                    log.debug("SyncGroup failed: {}, marking coordinator unknown", error.message());
+                    log.debug("SyncGroup failed: {} Marking coordinator unknown. Sent generation was {}",
```
Comment (guozhangwang) [inline clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java]: Ditto here, if we think we should pay attention to any errors excluding things like coordinator loading in progress let's just make them all info.

**Review by guozhangwang — COMMENTED** [review]
A few comments.

File: clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java (hunk 2 above)
Comment (ableegoldman) [reply clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java]: Sounds good

**Pull request merged**